generated/nodejs/google-cloud-node/packages/errorreporting/src/v1beta1/index.js

```diff
@@ -1,5 +1,5 @@
-/*!
- * Copyright 2016 Google Inc. All Rights Reserved.
+/*
+ * Copyright 2016 Google Inc. All rights reserved.
```

Comment:
This is funny, I didn't realized there was difference among capital R and r in our templates 😉 

But, it seems like, hand-written code in `google-cloud-node` uses capital R while GAPIC code uses small r. Tiny, but probably we want to update the template to use capital R.